### PR TITLE
projects: ad9265: add IIO support

### PR DESCRIPTION
**Pull request by amiclaus**
Allow the user to read/write via UART using IIO Oscilloscope.

Signed-off-by: Antoniu Miclaus <antoniu.miclaus@analog.com>

### CODE REVIEW

**Review by amiclaus — COMMENTED** [review]
V2:
- fix doxygen

File: projects/ad9265-fmc-125ebz/src.mk [hunk 1]
```diff
@@ -10,22 +10,47 @@
...
+ifeq (y,$(strip $(TINYIIOD)))
+INCS +=	$(INCLUDE)/xml.h						\
+	$(INCLUDE)/fifo.h						\
+	$(INCLUDE)/irq.h						\
+	$(INCLUDE)/uart.h						\
+	$(INCLUDE)/list.h						\
+	$(PLATFORM_DRIVERS)/irq_extra.h					\
+	$(PLATFORM_DRIVERS)/uart_extra.h                                \
+	$(NO-OS)/iio/iio_axi_adc/iio_axi_adc.h
+endif
```
Comment (adrimbarean) [inline projects/ad9265-fmc-125ebz/src.mk:56]: nitpick: files should have newline as endings

File: projects/ad9265-fmc-125ebz/src/app_iio.c [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+/******************************************************************************/
+static struct irq_ctrl_desc *irq_desc;
+
+struct iio_data_buffer g_read_buff = {
+	.buff = (void *)ADC_DDR_BASEADDR,
+	.size = 0xFFFFFFFF,
+};
+
+/**
+ * @brief Application IIO setup.
```
Comment (adrimbarean) [inline projects/ad9265-fmc-125ebz/src/app_iio.c:64]: Function parameter should be included in comment.

**Review by amiclaus — COMMENTED** [review]
V2:
- add function parameter header comment
- add new line at end of file

**Review by adrimbarean — APPROVED** [review]
Looks good to me.

**Pull request merged**